### DISCUSSION TITLE
Specify Revisions in Project Template

### DISCUSSION THREAD

**Discussion by zachhutchins** [mia-platform/community#480] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

1.98.6

### Description

Looking at previous discussions and your [documentation](https://docs.mia-platform.eu/docs/development_suite/company/project-templates#create-default-services) I do not see anything that references how one would automatically establish different revisions.

Wondering if this is possible? If so, how would one go about configurating that in their project template.

### Actual Outcome

I have a need to automatically establish revisions in a project template, these revisions are to control different environments, which all do not necessarily have the same setup.

### Expected Outcome

I cannot seem to find a way to set up revisions in our project template.

**Comment by galatifederico** [comment, marked as answer]
Hi @zachhutchins, at the moment is not possible to define revisions in project templates.
